packages/cli/src/handlers/dbt/run.ts

```diff
@@ -53,7 +54,7 @@ export const dbtRunHandler = async (
     }
     await generateHandler({
         ...options,
-        assumeYes: true,
+        assumeYes: options.assumeYes,
```

Comment:
We can remove the line as it is passed in `...options,`

Reply:
ty, done here: https://github.com/lightdash/lightdash/pull/11073/commits/868df8f55bf6fdef51a98ac7d0e8947fb8e5d93f